hack/README.md

```diff
@@ -5,3 +5,5 @@ This package contains a various scripts that are used by Helm Broker developers.
 ## Purpose
 
 This directory contains tools, such as Go fmt, Go lint, and Go vet, that help to maintain the source code compliant to Go best coding practices. It also includes utility scripts that generate code, and scripts executed on CI pipelines.
+
+The `release` directory contains the scripts which are essential for the release process.
```

Comment:
I'd add another section called `## Project structure` with the directory tree, such as here:
https://github.com/kyma-project/kyma#project-structure

Please also fix the first sentence of this document:
This package contains ~~a~~ various scripts that are used by Helm Broker developers.